cflat.md

```diff
@@ -0,0 +1,100 @@
+> [name=Arbie, Yin, Guo, Ashima]
```

Comment:
This is not valid Markdown.

I don't think it's important enough to put at the top of the directory tree. If we need this documentation, it should go in the `docs` direcotry with the other main documentation.